### ISSUE TITLE
What is going on here in nbodykit_survey.py?

### ISSUE THREAD

**Issue by pmcdonal** [adematti/cosmopipe#3]
I edited it to put in zeff, so... not comfortable not understanding what I did... 
See comment "what is going on here with the multiplication when there is more than one catalog"

https://github.com/adematti/cosmopipe/blob/6564c973459d8783c6083eaf5bbab39cdcecc686/cosmopipe/estimators/power_spectrum/nbodykit_survey.py#L72-L86

**Comment by adematti** [comment]
Multiple catalogs are only used for cross-correlations, so the actual question is: how to define an effective redshift for a cross-correlation? In eBOSS we have followed https://arxiv.org/pdf/2007.09011.pdf, e.q. 8, that extends straightforwardly to cross-correlations.

**Comment by pmcdonal** [comment]
I didn't mean with zeff - I put that in - I mean with what was there, i.e., with the *product* computing wdata2 within the loop. The problem is, when reading this from my point of view, just trying to get going with a simple power spectrum, I have no reason to  know when/why there might be more than one catalog. E.g., aren't there actually 2 when the randoms are fed in to compute window? (i.e., like 2 copies of the randoms - I haven't figured out the reason for that)
About the zeff though: yes, I'm not surprised you have a formula like that in the paper - I just put in the simplest thing I could do to get going.

**Comment by adematti** [comment]
Yes, there are 2 when computing the window, as the last one receives the 1/distance^n of the wide-order development (in https://github.com/adematti/cosmopipe/blob/e0ae22b434c415f7fb8028c26276b52974a539ee/cosmopipe/estimators/window_function/fft_window_function.py#L116 - the reason I am changing weight_fkp is because nbodykit tolerates two input catalogs only if they have the same positions and weight_comp - should be changed when revisiting nbodykit's algorithm).
Leaving open til I write some comments.
